A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg

```diff
@@ -192,11 +192,11 @@ spoiler_extension_time = 2 ; Spoilers extension time (SECONDS)
 spoiler_handle_available = 1 ; Spoiler handles available true/false
 auto_spoiler_available = 1 ; Auto spoilers available true/false
 auto_spoiler_min_speed = 72 ; Auto spoilers minimum trigger speed
-positive_g_limit_flaps_up = 2.4 ; Flap positive load limit when up. Same dimension as gravity vector FEET/SECONDS^2
-positive_g_limit_flaps_down = 1.68 ; Flap positive load limit when down. Same dimension as gravity vector FEET/SECONDS^2
-negative_g_limit_flaps_up = -1.92 ; Flap negative load limit when up. Same dimension as gravity vector FEET/SECONDS^2
-negative_g_limit_flaps_down = -1.44 ; Flap negative load limit when down. Same dimension as gravity vector FEET/SECONDS^2
-load_safety_factor = 1 ; Flap negative load limit when down. Same dimension as gravity vector FEET/SECONDS^2
+positive_g_limit_flaps_up = 2.5 ; Flap positive load limit when up. Same dimension as gravity vector FEET/SECONDS^2
```

Comment:
If these are actually feet/seconds^2 as indicated they'd be quite a bit higher than you've got here (1g = 9.8 m/s2 = 32 ft/s2). Seems the unit in these comments is bogus?

Reply:
The units are correct, but not really relevant to the issue.  Why do you think they are bogus?